cdap-docs/application-templates/source/etl/templates/sinks/batch/cube.rst

```diff
@@ -17,3 +17,33 @@ sink.
 To configure transformation from a StructuredRecord to a CubeFact, the mapping
 configuration is required, as following the StructuredRecordToCubeFact documentation.
 
+.. rubric:: Use Case
+
+TODO: Fill me out
```

Comment:
Are these going to be completed before the release? If not, perhaps they should say "TODO: To be completed in a later release."